.github/workflows/test-build-and-deploy.yml

```diff
@@ -78,6 +78,7 @@ jobs:
           flags: unittests # optional
           fail_ci_if_error: true # optional (default = false)
           verbose: true # optional (default = false)
+          range: 40..80
```

Comment:
I dont see this argument for the github action
https://github.com/codecov/codecov-action#arguments
Where did you find this?

Reply:
yep fixed !